oktapam/constants/descriptions/attributes.go

```diff
@@ -93,6 +93,7 @@ var (
 	NextUnixUID                               = "The UID to use when creating a new Server User. Default value starts at 60001."
 	OIDCIssuerURL                             = "The OIDC Issuer URL to use when configuring your Kubernetes cluster. "
 	OSAttribute                               = "The AD attribute that defines the operating system of discovered servers."
+	ParentFolder                              = "The secret folder which should contain this entity."
```

Comment:
```suggestion
	ParentFolder                              = "The secret folder which contains this entity."
```

Reply:
Is the ParentFolder just the normal directory that contains the oktapam_secret_folder, or is there something special about it?
If it's just a normal directory, consider changing this string to:
```suggestion
	ParentFolder                              = "The directory which contains this entity."
```

Reply:
Also, is "this entity" = the oktapam_secret_folder? If so, we might want to call that out a little more.